torch_xla_py/data_parallel.py

```diff
@@ -200,6 +200,10 @@ def __init__(self, network, device_ids=None, batchdim=0, drop_last=False):
       self._contexts.append(Context(torch.device(device)))
       self._native_run = True
 
+  @property
+  def models_dict(self):
+    return dict(zip(self._device_ids, self._models))
```

Comment:
I'd just have a models() function that returns a list of tuples (IOW drop the dict()).